test/e2e/tests/onboarding/onboarding.spec.ts

```diff
@@ -300,7 +317,9 @@ describe('MetaMask onboarding', function () {
     );
   });
 
-  it('Provides an onboarding path for a user who has restored their account from state persistence failure', async function () {
+  // But #38077 - After estoring account from state persistence failure Metamask unlock is not working
+  // eslint-disable-next-line mocha/no-skipped-tests
+  it.skip('Provides an onboarding path for a user who has restored their account from state persistence failure', async function () {
```

Comment:
To be reworked in a follow-up PR:
- But -> Bug
- Investigate if the accounts-framework states are valid during this scenario?
- Try to re-enable this test